{{cookiecutter.project_name}}/.github/workflows/ci-pybind11.yml

```diff
@@ -60,3 +60,6 @@ jobs:
 
       - name: Test package
         run: python -m pytest -ra
+
+      - name: Upload coverage report
+        uses: codecov/codecov-action@v3.1.0
```

Comment:
I have updated the `noxfile`, but uploading the coverage report here makes no sense until tests are run using `nox`. Should I make the required changes?